samples/powerstore_v260_ops_410.yaml

```diff
@@ -5,16 +5,17 @@ metadata:
   namespace: test-powerstore
 spec:
   driver:
-    # Config version for CSI PowerStore v2.3.0 driver
-    configVersion: v2.3.0
+    # Config version for CSI PowerStore v2.6.0 driver
+    configVersion: v2.6.0
     # Controller count
     replicas: 2
     dnsPolicy: ClusterFirstWithHostNet
     forceUpdate: false
     fsGroupPolicy: ReadWriteOnceWithFSType
+    storageCapacity: true
```

Comment:
This can be removed, as OpenShift 4.10 uses k8s v1.23

Reply:
Addressed.
Thank You.!